### PR TITLE
Add a basic test

### PR DESCRIPTION
**Pull request by mattprecious**
Start setting up the testing APIs.

### CODE REVIEW

**Comment by mattprecious** [comment]
Ugh, bitten by my own laziness: `Caused by: java.lang.IllegalStateException: Unsupported OS: linux`. Will fix that and circle back.

File: build.gradle.kts [hunk 1]
```diff
@@ -7,6 +7,8 @@ plugins {
 	alias(libs.plugins.sqldelight)
 	alias(libs.plugins.kotlin.jvm)
 	alias(libs.plugins.kotlin.plugin.serialization)
+
+	id("java-test-fixtures")
```
Comment (JakeWharton) [inline build.gradle.kts:11]: Not needed? It's hard to see on mobile, but aren't the tests within this module?

**Review by JakeWharton — APPROVED** [review]

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -69,3 +69,10 @@ jobs:
           path: native
 
       - run: ./gradlew build
+
+      - uses: actions/upload-artifact@v3
+        if: failure()
```
[the remark this answers is not in the document]
Comment (mattprecious) [reply .github/workflows/build.yaml:74]: I forget why I did this. I popped this out of a really old stash. Does it exist when tests pass?

File: build.gradle.kts (hunk 1 above)
Comment (mattprecious) [reply build.gradle.kts:11]: It's completely unnecessary, but text fixtures are cool and I finally have a project where I can use them (unnecessarily).

File: .github/workflows/build.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (JakeWharton) [reply .github/workflows/build.yaml:74]: Yep

**Pull request merged**